doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,153 @@
+{
+    // Info displayed in the Extension Store
+    "id": "postgres-vs",
+    "name": "PostgreSQL Virtual Schema",
+    "description": "...",
+    "version": "2.0.1",
+    // Dependencies
+    "bucketFsUpload": {
+        "jdbcDriver": {
+            // Download URL for the JDBC driver, e.g. Maven Central URL
+            "downloadUrl": "https://jdbc.postgresql.org/download/postgresql-42.3.3.jar",
+            // License URL
+            "licenseUrl": "https://github.com/pgjdbc/pgjdbc/blob/master/LICENSE",
+            // True if the user must agree to the license before installation, e.g. for Oracle JDBC driver
+            "licenseAgreementRequired": false,
+            // Path and filename when uploading files to BucketFS
+            "bucketFsPath": "/buckets/${BFS_SERVICE}/${BUCKET}/virtual-schema/postgresql-jdbc-driver.jar"
```

Comment:
so, you assume that you directly copy the file into the file system, not sure if this is a good idea. It, generally, works but is a bit trickly.

Reply:
I have no idea what are the potential problems. Which alternative do you propose?

Reply:
Use the http interface of the bucketfs. The usual problems are wrong user or permissions or copies in progress trip the bucketfs.